src/providers/s3Provider.ts

```diff
@@ -1,4 +1,5 @@
 import { ListObjectsV2Command, S3Client } from '@aws-sdk/client-s3';
+import { FetchHttpHandler } from '@smithy/fetch-http-handler';
```

Comment:
What's this? 👀

Reply:
Uses the global `fetch` impl in workers instead of the polyfill it uses by default, allows for mocking